lang/en/local_smartmedia.php

```diff
@@ -171,8 +171,6 @@
 $string['settings:enrichment:transcribe_help'] = 'Attempt an automated transcription on audio and video files.';
 $string['settings:ffprobe:pathtoffprobe'] = 'FFProbe binary path';
 $string['settings:ffprobe:pathtoffprobe_desc'] = 'The path to the FFProbe binary on the server running Moodle.';
-$string['settings:useproxy'] = 'Use proxy';
-$string['settings:useproxy_help'] = 'Smartmedia can use configured Moodle proxy to reach AWS API';
```

Comment:
Sentences for descriptions / non-titles should probably end with a full-stop.

Reply:
These are removals, ive killed the setting entirely